packages/elements/src/tree/__test__/tree.test.js

```diff
@@ -267,7 +267,6 @@ describe('tree/Tree', () => {
       event = await oneEvent(el, 'expanded-changed');
       expect(event.detail.value, 'Group should be expanded').to.be.true;
       expect(event.detail.item, 'Item should be the same as the original').to.equal(nestedData[0]);
-      el.dispatchEvent(keyArrowDown);
```

Comment:
Why remove this event?

Reply:
![image](https://user-images.githubusercontent.com/86233706/179189525-64f2a0ac-726e-48cf-aef7-96350311a362.png)

So if you look at the testing data, you will see that only first item is expandable. If arrow down is pressed, it will move into item that has no children so cannot expand. It was a false test case.

Reply:
OK